packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+
+    protected override willUpdate(
+        changedProperties: PropertyValueMap<this>
+    ): void {
+        this._setLocale();
+
+        if (changedProperties.has('selectedDate')) {
+            this._setCurrentCalendarDate();
+        }
+
+        if (changedProperties.has('min')) {
+            this._setMinCalendarDate();
+        }
+
+        if (changedProperties.has('max')) {
+            this._setMaxCalendarDate();
+        }
+    }
+
+    protected override render(): TemplateResult {
+        return when(
+            this._currentDate,
+            () => html`
+                ${this.renderCalendarHeader()}${this.renderCalendarGrid()}
+            `
+        );
+    }
+
+    public renderCalendarHeader(): TemplateResult {
+        const monthAndYear = this._capitalFirstLetter(
+            this._formatDate(this._currentDate, {
+                month: 'long',
+                year: 'numeric',
+            })
+        );
+
+        return html`
+            <div class="spectrum-Calendar-header">
+                <!--
+                 * TODO: Attribute 'role="heading"' removed, due to error 'The "heading" role requires the attribute
+                 * "aria-level"'
+                -->
+                <div
+                    class="spectrum-Calendar-title"
+                    aria-live="assertive"
+                    aria-atomic="true"
+                >
+                    ${monthAndYear}
+                </div>
+
+                <!-- TODO: Translate "Previous" -->
+                <sp-action-button
+                    quiet
+                    size="s"
+                    aria-label="Previous"
+                    title="Previous"
+                    class="spectrum-ActionButton spectrum-Calendar-prevMonth"
+                    ?disabled=${this.disabled}
+                    @click=${this.handlePreviousMonth}
+                >
+                    <div slot="icon">
+                        <slot name="prev-icon">
+                            <sp-icon-chevron-left></sp-icon-chevron-left>
+                        </slot>
+                    </div>
+                </sp-action-button>
+
+                <!-- TODO: Translate "Next" -->
+                <sp-action-button
+                    quiet
+                    size="s"
+                    aria-label="Next"
+                    title="Next"
+                    class="spectrum-ActionButton spectrum-Calendar-nextMonth"
+                    ?disabled=${this.disabled}
+                    @click=${this.handleNextMonth}
+                >
+                    <div slot="icon">
+                        <slot name="prev-icon">
+                            <sp-icon-chevron-right></sp-icon-chevron-right>
+                        </slot>
+                    </div>
+                </sp-action-button>
+            </div>
+        `;
+    }
+
+    public renderCalendarGrid(): TemplateResult {
+        return html`
+            <!-- TODO: Implement keyboard navigation -->
```

Comment:
See https://opensource.adobe.com/spectrum-web-components/tools/roving-tab-index/

Reply:
Might be some things to add beyond that to really fill out the expectations here, but a good start.

Reply:
As I mentioned earlier, all TODOs will be resolved when the new date/time picker (which includes this calendar component) is ready to be merged into the main branch. All pending issues are being listed in the respective READMEs of each component